This patch resolves some internal test suite failures and has been
taken from the community. Details at
https://sourceforge.net/p/gimp-print/mailman/message/36038601/
diff --git a/src/cups/genppd.c b/src/cups/genppd.c
index 2e47f34c..ad44b428 100644
--- a/src/cups/genppd.c
+++ b/src/cups/genppd.c
@@ -1277,6 +1277,7 @@ print_page_sizes(gpFile fp, stp_vars_t *v, int simplified,
 	  continue;
 	}
       if (simplified && num_opts >= 10 &&
+	  (!desc.deflt.str || strcmp(opt->name, desc.deflt.str) != 0) &&
 	  (papersize->paper_unit == PAPERSIZE_ENGLISH_EXTENDED ||
 	   papersize->paper_unit == PAPERSIZE_METRIC_EXTENDED))
 	continue;
